pages/avatar.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css"
+    />
+    <link rel="stylesheet" href="/styles.css" />
+    <title>Dazzle : Avatar</title>
+  </head>
+  <body>
+    <div class="grid-layout">
+      <nav></nav>
+      <sidebar class="sidebar scrollbar"></sidebar>
+
+      <div class="main">
+        <div class="component-heading">
+          <h2>Avatar</h2>
+          <p>Avatars are used to represent user initials or user profile pictures. They help in visually representing user identity. </p>
+        </div>
+
+        <h4>Rounded Avatar</h4>
+        <p>Avatars are available in 3 sizes: <span class="secondary-text">size-lg, size-md and size-sm</span>. For representing user avatar in circular shape, just add a class <span class="code secondary-text">rounded</span> to the <span class="code secondary-text">img</span> tag. </p>
+        <div class="component-example-container">
+          <img
+            class="avatar rounded size-lg"
+            src="/images/profile_picture3.jpeg"
+          />
+          <img
+            class="avatar rounded size-md"
+            src="/images/profile_picture2.jpeg"
+          />
+          <img
+            class="avatar rounded size-sm"
+            src="/images/profile_picture4.jpeg"
+          />
+        </div>
+
+        <pre class="component-code-snippet">
+          <code class="language-html">
+            <img class="avatar rounded size-lg" src="/images/profile_picture3.jpeg"/>
+
+            <img class="avatar rounded size-md" src="/images/profile_picture2.jpeg"/>
+
+            <img class="avatar rounded size-sm"src="/images/profile_picture4.jpeg"/>
+          </code>
+        </pre>
+
```

Comment:
Just a suggestion, as before starting a new component you are using `<h4>` and `<p>`, and then the component. So, you can wrap that whole component in a `<section>` I think that will increase the readability, that this is a new section.

Reply:
Hey, I have made the changes. Thanks!